BUILDGUIDE.md

```diff
@@ -200,9 +200,9 @@ Tests can be built and run with custom Target Frameworks. See the below examples
 ### Building Tests:
 
 ```bash
-> msbuild /t:BuildTestsNetFx /p:TargetNetFxVersion=net461
+> msbuild /t:BuildTestsNetFx /p:TargetNetFxVersion=net462
```

Comment:
Any reason we bump to 462 for the tests?

Reply:
This is the sample of `TargetNetFxVersion` since our default is 461 right now, the sample shows a version other than the default. previously, when we had 46 as default and it was 461 showing a different version as a different version.

Reply:
I see. Thanks.